std_msgs/Header header
float64 odom_x
float64 odom_y
float64 odom_theta
float64 meas_rho
float64 meas_beta
float64 meas_theta
float64 delta_t
float64 gt_rho
float64 gt_beta
float64 gt_theta
float64 case0_x
float64 case0_y
float64 case0_t
float64 case1_x
float64 case1_y
float64 case1_t
float64 case2_x
float64 case2_y
float64 case2_t
float64 case3_x
float64 case3_y
float64 case3_t
float64 case4_x
float64 case4_y
float64 case4_t
float64 case4_vj
float64 case4_wj
